nautobot/extras/utils.py

```diff
@@ -269,6 +269,7 @@ def refresh_job_model_from_job_class(job_model_class, job_source, job_class, *,
             "slug": default_slug[:JOB_MAX_SLUG_LENGTH],
             "grouping": job_class.grouping[:JOB_MAX_GROUPING_LENGTH],
             "name": job_class.name[:JOB_MAX_NAME_LENGTH],
+            "is_job_hook_receiver": getattr(job_class, "is_job_hook_receiver", False),
```

Comment:
Would it make more sense to do something like `issubclass(job_class, JobHookReceiver)` and not need a separate class attribute? Or does that get us into trouble with circular imports or such?

Reply:
Could probably do something like this: https://github.com/nautobot/nautobot/blob/develop/nautobot/extras/utils.py#L191

Reply:
> Could probably do something like this: https://github.com/nautobot/nautobot/blob/develop/nautobot/extras/utils.py#L191

This is what I did initially but I'm seeing these workarounds for circular dependencies in a lot of places and didn't want to contribute to more tech debt. I think it might be a better idea to use this as an opportunity to clean some of that up like @jathanism suggested in #204.